reference/docs-conceptual/community/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,112 @@
+  to @mention the reviewer. For example, if the change is minor and you don't need a full technical
+  review, explain this in a comment.
+
+## PR Process steps
+
+1. Writer: Create PR
+   - Link any issues resolved by the PR
+   - Use GitHub's [autoclose](https://help.github.com/en/articles/closing-issues-using-keywords)
+     feature to close the issue
+1. Writer: Assign peer reviewer
+1. Reviewer: proofreads and comments (as necessary)
+1. Writer: Incorporate review feedback
+1. Both: Review preview rendering
+1. Both: Review validation report - fix warnings and errors
+1. Writer: Add sign-off comment (include Acrolinx info)
+1. Reviewer: Mark review "Approved"
+1. Repo Admin: Merge PR (see below for criteria)
+
+## Content Reviewer Checklist
+
+See the [editorial checklist](editorial-checklist.md) for a more comprehensive list.
+
+- Proofread for grammar, style, concision, technical accuracy
+- Ensure examples still apply for the target version
+- Check Preview rendering
+- Check metadata - ms.date, remove ms.assetid, ensure required fields
+- Validate markdown correctness
+  - See style guide for formatting specific content
+- Reorganize examples as follows:
+  - Intro sentence(s)
+  - Code and output
+  - Detailed explanation of code (as necessary)
+- Check hyperlinks for accuracy
+  - Replace or remove TechNet/MSDN links
+  - Ensure minimum number of redirects to target
+  - Ensure HTTPS
+  - Correct link type
+    - File links for local files
+    - URL links for files outside of the docset
+  - Remove locales from URLs
+  - Simplify URLs pointing to `docs.microsoft.com`
+
+## Branch Merge Process
+
+The staging branch is the only branch that should ever be merged into live. Merges from short-lived
+(working) branches should be squashed.
+
+| *Merge from/to*  | *release-branch* | *staging*        | *live*      |
+| ---------------- |:----------------:|:----------------:|:-----------:|
+| *working-branch* | squash and merge | squash and merge | Not allowed |
+| *release-branch* | &mdash;          | merge            | Not allowed |
+| *staging*        | rebase           | &mdash;          | merge       |
+
+### PR Merger checklist
+
+- Content review complete
+- Correct target branch for the change
+- No merge conflicts
+- All validation and build step pass
+  - Warnings and suggestions should be fixed (see [Notes](#notes) for exceptions)
+  - No broken links
+- Merge according to table
+
+### Notes
+
+The following warnings can be ignored:
+
+- > Can't find service name for `<version>/<modulepath>/About/About.md`
+
+- > Metadata with following name(s) are not allowed to be set in Yaml header, or as file level
+  > metadata in docfx.json, or as global metadata in docfx.json: `locale`. They are generated by
+  > Docs platform, so the values set in these 3 places will be ignored. Please remove them from all
+  > 3 places to resolve the warning.
```

Comment:
Would these be better in a codefence than a note bubble? The render looks odd (partial bubble off a line item).

Reply:
Fixed